packages/lib/src/spatial-navigation/components/FocusableView.tsx

```diff
@@ -0,0 +1,25 @@
+import { SpatialNavigationNode, SpatialNavigationNodeDefaultProps } from './Node';
+import { View } from 'react-native';
+import { forwardRef } from 'react';
+import { SpatialNavigationNodeRef } from '../types/SpatialNavigationNodeRef';
+
+type Props = SpatialNavigationNodeDefaultProps & {
+  children:
+    | React.ReactElement
+    | ((props: { isFocused: boolean; isActive: boolean }) => React.ReactElement);
+};
+
+export const SpatialNavigationFocusableView = forwardRef<SpatialNavigationNodeRef, Props>(
+  ({ children, ...props }: Props, ref) => {
+    return (
+      <SpatialNavigationNode isFocusable {...props} ref={ref}>
+        {({ isFocused, isActive }) => (
+          <View>
```

Comment:
You should definitely pass an option `style` property ;)

Reply:
True !